libs/schedulers/include/hpx/schedulers/shared_priority_queue_scheduler.hpp

```diff
@@ -427,27 +433,39 @@ namespace hpx { namespace threads { namespace policies {
                                 std::to_string(data.schedulehint.mode));
                     }
                     // we do not allow threads created on other queues to 'run now'
-                    // as this caues cross-thread allocations and map accesses
+                    // as this causes cross-thread allocations and map accesses
                     if (local_num != thread_num)
                     {
                         run_now = false;
-                        spq_deb.debug(debug::str<>("create_thread"), "pool",
-                            parent_pool_->get_pool_name(), "hint", msg, "dest",
-                            "D", debug::dec<2>(domain_num), "Q",
-                            debug::dec<3>(q_index), "this", "D",
-                            debug::dec<2>(d_lookup_[local_num]), "Q",
-                            debug::dec<3>(local_num), "run_now OVERRIDE ",
-                            run_now, debug::threadinfo<thread_init_data>(data));
+                        // clang-format off
+                        spq_deb.debug(debug::str<>("create_thread")
+                            , "pool", parent_pool_->get_pool_name()
+                            , "hint", msg
+                            , "dest"
+                            , "D", debug::dec<2>(domain_num)
+                            , "Q", debug::dec<3>(q_index)
+                            , "this"
+                            , "D", debug::dec<2>(d_lookup_[local_num])
```

Comment:
This will horribly fail if `local_num == -1` (that's a valid possibility, as it is initialized by `local_thread_number()` which may return `-1`). 

I'm aware that this is not a problem introduced by this PR, I just thought to mention it anyways.